WeRadiate-ThermoSense.ino

```diff
@@ -289,17 +291,35 @@ void setup_platform(void)
                 flags = 0;
                 }
 
+	/* initialize the FLASH */
+	if (gFlash.begin(&gSPI2, Catena::PIN_SPI2_FLASH_SS))
+		{
+		fFlash = true;
+		gFlash.powerDown();
+		gCatena.SafePrintf("FLASH found, put power down\n");
+		}
+	else
+		{
+		fFlash = false;
+		gFlash.end();
+		gSPI2.end();
+		gCatena.SafePrintf("No FLASH found: check wiring\n");
```

Comment:
change "check wiring" to "check board", because if the flash isn't working, the board is broken.